net.sourceforge.xournal.json

```diff
@@ -0,0 +1,114 @@
+{
+  "app-id": "net.sourceforge.xournal",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.28",
+  "sdk": "org.gnome.Sdk",
+  "command": "xournal",
+  "rename-icon": "xournal",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--filesystem=host"
```

Comment:
apologies if this has been asked before - but what do you need full host filesystem permissions for? That seems like a wide scope that should be limited if possible.

Reply:
Users should be able to open and save to any files from the filesystem right?